integration/simple-snake/google/protobuf/struct.ts

```diff
@@ -61,8 +61,8 @@ export interface Struct_FieldsEntry {
 /**
  * `Value` represents a dynamically typed value which can be either
  * null, a number, a string, a boolean, a recursive struct value, or a
- * list of values. A producer of value is expected to set one of these
- * variants. Absence of any variant indicates an error.
+ * list of values. A producer of value is expected to set one of that
+ * variants, absence of any variant indicates an error.
```

Comment:
I didn't make a change here, I suspect the original fix didn't update the integration test output?

Reply:
@zakhenry apologies for the late reply here; these changes are b/c `protoc` has different versions of its `struct.proto` / etc. files based on which version of `protoc` you have installed natively on your machine.

We used to have these "well, I have protoc x.y, you have protoc y.z" issues all the time, but recently we moved to a docker-based setup that, if you use the `npm run` scripts that are docker-based, should always use the same version of `protoc` to a) generate the `.bin` files, which should then b) mean you shouldn't have any weird "just comment" diffs in these files.

Can you try that and see if that clears up these other changes?

Thanks!

Reply:
ah yep that did the trick, thanks!